hackathonbaobab2020/__init__.py

```diff
@@ -1,3 +1,31 @@
+from typing import List, Dict
 from .core import *
 from .execution import *
 from .solver import *
+from .solver import solvers
```

Comment:
Does this two imports contradict each other?

Reply:
for some reason there was another `solvers` variable. I do not like importing *. I think I will import things explicitly.